sdk/appservice/arm-appservice-profile-2020-09-01-hybrid/package.json

```diff
@@ -109,5 +99,45 @@
     ],
     "disableDocsMs": true,
     "apiRefLink": "https://docs.microsoft.com/javascript/api/@azure/arm-appservice-profile-2020-09-01-hybrid?view=azure-node-preview"
+  },
+  "type": "module",
+  "tshy": {
+    "project": "./tsconfig.src.json",
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
```

Comment:
Are we shipping for React-Native and Browser support?  If not, please remove the `esmDialects`.